### ISSUE TITLE
Consistency with ModelMuse

### ISSUE THREAD

**Issue by rogiersbart** [rogiersbart/RMODFLOW#11]
Let's create a place to also discuss this. My point of view at the moment is that RMODFLOW should follow ModelMuse conventions as closely as possible. Main argument for that is that the package should in this case be a more welcoming environment for the ModelMuse user. Given the GUI aspect, I think it is still more likely people are introduced to MODFLOW through ModelMuse than through RMODFLOW, FloPy or mfLab for instance.
There's two sides to this I think:
1. File names: It is very useful we adopt the file extension conventions from ModelMuse I think (*i.e.* for our function names), as it simplifies figuring out what functions have to be used to handle specific files. I mentioned in #8 we should aim at keeping the rmf_read_fhd and rmf_read_bhd functions for that reason. I would even go further and suggest to remove rmf_read_hed for instance, and replace it by one of these two. There's no reason for introducing a separate RMODFLOW function name not based on any file extension convention.
2. Functionality: Certainly at this stage, RMODFLOW should not aim at doing things that are not part of the ModelMuse scope. So standard calibration through optim based on pval parameters should be there I think. More complex methods (based on pval parameters, or with custom parameters set up in R) should be moved to an extension package.

**Comment by rogiersbart** [comment]
Some more inconsistencies:
- pvl (current RMODFLOW) vs pval (ModelMuse)
- prj vs reference (also some variants of MODFLOW can store some of this info in regular files)

**Comment by cneyens** [comment]
Agreed. I've tried to keep consistency with the [MODFLOW-2005 online manual](https://water.usgs.gov/ogw/modflow/MODFLOW-2005-Guide/) as much as possible. Since there are no conventions for output, the following classes are used in RMODFLOW:

- `hed` for simulated heads
- `ddn` for simulated drawdown
- `cbc` for cell-by-cell flow budget
- `bud` for the volumetric budget as printed to the listing file
- `hpr` for the HOB output file

These can all be changed without too much effort (and would involve changing the function names and calls, class names and corresponding methods). Same goes for `pvl`.

That being said, I would like to make a case for keeping `rmf_read_head` and `rmf_read_drawdown`. ModelMuse conventions themselves are also arbitrary and not everybody is familiar with them. `rmf_read_head` is clear to **all** MODFLOW users. `rmf_read_fhd` is only clear to ModelMuse users. Programatically, `rmf_read_head` is the parent function of `rmf_read_fhd/bhd` (which are just wrappers with the binary argument set differently).

Therefore I propose the following:
- Rename `rmf_read_hed` to `rmf_read_head`. Similary `ddn` --> `drawdown`
- Have `rmf_read_fhd/bhd` as they are now: i.e. wrappers for `rmf_read_head` (similarly for drawdown)
- Change classes `hed` and `ddn` to `head` and `drawdown`

Regarding functionality; I fully agree. I do think we should show how to set up an objective function for calibration using RMODFLOW (perhaps in a vignette) though. This is not too hard and it gives users something to go by when `rmf_optimize` doesn't work. Especially since not all MODFLOW models have pval parameters (and USGS is even moving away from that: FloPy doesn't support it I believe and MF6 doesn't either)

Regarding prj vs reference: I have been working on [spatial conversions](https://github.com/cneyens/RMODFLOW/tree/feature/spatial) (sf/raster/stars --> RMODFLOW classes and vice versa). Here I've consistently used prj as it is now. As briefly discussed in #4 , prj would be part of the `dis` object. It would like to continue the work using `prj` since it works as is and if we want to change, do it after this feature is complete.
